views/tables/ParticipantsTable/index.jsx

```diff
@@ -64,29 +64,37 @@ const ParticipantsTable = (props) => {
   const cellRenderer = (participant, property) => {
     switch (property) {
       case 'synced':
-        const isToday = dayjs(participant[property]).isToday()
-        const isYesterday = dayjs(participant[property]).isYesterday()
-        const isTodayOrYesterday = isToday || isYesterday
-        return (
-          <Typography
-            sx={{
-              color: isTodayOrYesterday ? 'green.500' : 'text.primary',
-            }}
-          >
-            {isToday
-              ? 'Today'
-              : isYesterday
-              ? 'Yesterday'
-              : participant[property]
-              ? dayjs(participant[property]).format('MMM-D-YYYY')
-              : ''}
-          </Typography>
-        )
+        const participantSyncedDate = participant[property]
+
+        if (participantSyncedDate && dayjs(participantSyncedDate).isValid()) {
+          const isToday = dayjs(participantSyncedDate).isToday()
+          const isYesterday = dayjs(participantSyncedDate).isYesterday()
```

Comment:
we're still creating new dayjs objects multiple times for the same date.

```suggestion
        const participantSyncedDate = dayjs(participant[property])

        if (participant[property] && participantSyncedDate.isValid()) {
          const isToday = participantSyncedDate.isToday()
          const isYesterday = participantSyncedDate.isYesterday()
```

Reply:
[07f6094](https://github.com/AMP-SCZ/dpdash/pull/600/commits/07f609401a2ee7fdfef8fb09c31bc006d8746e20)